### PR TITLE
Add LDAP steps to Readme

### PR DESCRIPTION
**Pull request by rayyaoibm**

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
 # semanage fcontext -a -t container_file_t "<STORAGE-PATH>(/.*)?"
 # restorecon -Rv <STORAGE-PATH>
 ```
+Add a new section to https://github.com/IBM/Db2/tree/develop/deployment.
```
Comment (cg8) [inline deployment/README.md]: Did you intend to have this line?

File: deployment/README.md [hunk 2]
```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
...
+``` 
+oc get po | grep ldap | cut -d " " -f 1
+```
+Example output:
+```
+db2u-deployment-db2u-ldap-5cf745f7cf-ql6sh
+```
+## 2. Log in to the LDAP pod
+```
+oc rsh <LDAP-pod-name> /bin/bash
```
Comment (cg8) [inline deployment/README.md]: Could we do this in "1" command? Something like `oc rsh $(oc get po | grep ldap | cut -d " " -f 1) /bin/bash`

File: deployment/README.md [hunk 3]
```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
...
+```
+## 3. Find the next available LDAP user ID
+```
+source /opt/ibm/lib/utils.sh
+
+ldapPassword=$(get_ldap_root_password)
+
+echo $(($(ldapsearch -Z -H ldap:/// -D 'cn=bluldap,dc=blustratus,dc=com' -w "$ldapPassword" -b 'dc=blustratus,dc=com' '(objectClass=posixAccount)' | grep uidNumber | cut -d ":" -f 2 | sort -nr | head -n 1)+1))
```
Comment (cg8) [inline deployment/README.md:152]: Could we have a variable like LDAP_UID=....
Then, we only have to reference the variable in the script

File: deployment/README.md [hunk 4]
```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
...
+ldappasswd -x -Z -H ldap:/// -D "cn=bluldap,dc=blustratus,dc=com" -w "$ldapPassword" -S "uid=<user>,ou=People,dc=blustratus,dc=com" -s <password>
+```
+
+## 7. Verify the newly created LDAP user and credential
+a) Exit from the LDAP pod
+```
+exit
+```
+b) Log in to a db2u pod
+Example:
```
Comment (cg8) [inline deployment/README.md]: Can we skip "Example""? I don't think it is necessary

**Review by cg8 — CHANGES_REQUESTED** [review]

File: deployment/README.md (hunk 1 above)
Comment (rayyaoibm) [reply deployment/README.md]: Good catch. No, it's not intended. I'll remove this line.

File: deployment/README.md (hunk 2 above)
Comment (rayyaoibm) [reply deployment/README.md]: Good idea. Updated.

File: deployment/README.md (hunk 3 above)
Comment (rayyaoibm) [reply deployment/README.md:152]: In this case, the UID value needs to be manually edited inside the config file newuser.ldif. 

We plan to fully automate the setup (including auto generate the config file, etc) in the future.

File: deployment/README.md (hunk 4 above)
Comment (rayyaoibm) [reply deployment/README.md]: Remove the word "Example"

**Review by cg8 — APPROVED** [review]

**Pull request closed without merging**